pkg/image/resolve_test.go

```diff
@@ -274,9 +274,9 @@ func TestGetImages(t *testing.T) {
 	sourcesToAdd := getImageSourcesList(imagesSet)
 	linuxRKEImage := selectFirstEntry(linuxInfo.RKESystemImages).CoreDNS
 	linuxImagesOnly := append(
-		imagesToAdd,                         // from requirement
-		linuxRKEImage,                       // from system
-		"rancher/prom-alertmanager:v0.17.0", // from chart
+		imagesToAdd,                                    // from requirement
+		linuxRKEImage,                                  // from system
+		"rancher/prom-alertmanager:v0.17.0",            // from chart
```

Comment:
Did the formatter do this automatically?

Reply:
Yeah - I can change back if you want.

Reply:
It's fine if this is how the formatter had it.